build.sbt

```diff
@@ -291,6 +298,12 @@ lazy val unidocSettings = Seq(
     "-doc-title", "Delta Lake " + version.value.replaceAll("-SNAPSHOT", "") + " ScalaDoc"
   ),
 
+  ScalaUnidoc / unidoc / unidocAllSources := {
+    (ScalaUnidoc / unidoc / unidocAllSources).value
+      // io.delta.storage public APIs are Java-only
+      .map(_.filterNot(_.getCanonicalPath.contains("io/delta/storage")))
```

Comment:
this has to change .. right?

Reply:
yup I will be updating PR. thanks for checking.

Reply:
done